src/fsharp/tast.fs

```diff
@@ -3109,7 +3398,7 @@ and CcuResolutionResult =
     | UnresolvedCcu of string
 
 /// Represents the information saved in the assembly signature data resource for an F# assembly
-and PickledModuleInfo =
+and PickledCcuInfo =
```

Comment:
Maybe a glossary with all the acronyms can help. 
Easy doc and useful ( search  text `ccu` in repo -> glossary.md )